### ISSUE TITLE
Command not working:  sync the Output/REPL window namespace with the current file

### ISSUE THREAD

**Issue by LouDnl** [BetterThanTomorrow/calva#1503]
This command does not work at all. On Windows the command is mapped to ctrl+alt+c n.
`To sync the Output/REPL window namespace with the current file before switching, use the Switch Namespace of the Output/REPL Window to Current Namespace command, ctrl+alt+c alt+n.`

https://clojurians.slack.com/archives/CBE668G4R/p1643274711380319?thread_ts=1643273982.672629&cid=CBE668G4R

**Comment by joachimdraeger** [comment]
Same here. `Calva: Switch Namespace in Output/REPL Window to Current Namespace` doesn't do anything on Mac either.

**Comment by steffan-westcott** [comment]
The command does nothing on Linux either. Selecting the command from the palette or using the keyboard shortcut has no effect.

**Comment by PEZ** [comment]
We'll get to fixing this, promise! Meanwhile the workaround is to evaluate something in the file. Anything, doesn't matter what.

**Comment by PEZ** [comment]
Actually the command does exactly what it says it does, but then it forgets to tell you. 😄 

Less jokingly phrased. We did some changes to the output window recently around the prompt which led to that we don't print a prompt after this switch of namespace. If you evaluate something like `(def foo :foo)` in the repl window after using the command you will see that it actually is defined in the new namespace.

Anyway, it feels totally broken in the current state and I have just fixed it on the `dev` branch so a fix is soon in a VS Code editor near you.
